code/modules/clothing/suits/ego_gear/waw.dm

```diff
@@ -30,7 +30,7 @@
 	name = "armor sharpened with tears"
 	desc = "Tears fall like ash, embroidered as if they were constellations."
 	icon_state = "despair"
-	armor = list(RED_DAMAGE = 20, WHITE_DAMAGE = 20, BLACK_DAMAGE = 20, PALE_DAMAGE = 60)
+	armor = list(RED_DAMAGE = 20, WHITE_DAMAGE = 30, BLACK_DAMAGE = 20, PALE_DAMAGE = 60)
```

Comment:
Same here, why the buff?

Reply:
Well, its just that SST does not have a valuable enough main defense stat to justify it being worn atm. Pale defense is very useful, but it is rare, and not worth using over Lamp or Hornet because its 2/2/2/6 instead of lamp and hornet which are 6/3/3/3 and 3/3/6/3

Reply:
That's the entire point, this armor is pale-specific, "it is rare" doesn't apply when we have A LOT of pale-damaging abnormalities.